subsys/bluetooth/controller/Kconfig

```diff
@@ -560,6 +560,7 @@ config BT_CTLR_LE_POWER_CONTROL
 	bool "LE Power Control Request Feature"
 	depends on BT_CTLR_LE_POWER_CONTROL_SUPPORT
 	default y if BT_TRANSMIT_POWER_CONTROL
+	default y if BT_PATH_LOSS_MONITORING
```

Comment:
You should instead have `depends on BT_CTLR_LE_PATH_LOSS_MONITORING_SUPPORT && BT_CTLR_LE_POWER_CONTROL`  in `config BT_CTLR_LE_PATH_LOSS_MONITORING`, right?

Reply:
Yeah that is probably good to have as well. Worth having this `default y if BT_PATH_LOSS_MONITORING` here though, otherwise it just silently doesn't work.

Reply:
> otherwise it just silently doesn't work.

The Kconfig `depends on` should give build errors or warnings

Reply:
Yeah that's what I thought but without this change if I just enable  `CONFIG_BT_PATH_LOSS_MONITORING=y` then I get no warning about the missing dependency for `BT_CTLR_LE_PATH_LOSS_MONITORING`. However, if I just try and enable `CONFIG_BT_CTLR_LE_PATH_LOSS_MONITORING=y` directly, then I do get the warning.

Reply:
I presume this is for cases where `CONFIG_BT_CTLR=n` to not gives loads of warnings?

Reply:
> Yeah that's what I thought but without this change if I just enable `CONFIG_BT_PATH_LOSS_MONITORING=y` then I get no warning about the missing dependency for `BT_CTLR_LE_PATH_LOSS_MONITORING`. However, if I just try and enable `CONFIG_BT_CTLR_LE_PATH_LOSS_MONITORING=y` directly, then I do get the warning.

Is that the case with or without `default y if BT_PATH_LOSS_MONITORING`? If you have `default y if BT_PATH_LOSS_MONITORING` then you probably don't see any warnings, but it should show one without. Did you try a clean build?

Reply:
This is without `default y if BT_PATH_LOSS_MONITORING`, yes. And just double checked with a clean build

Reply:
Sounds odd. Did it get enabled in the .config file?

Reply:
No, not enabled in the .config

Reply:
Sounds strange that if it doesn't get enabled but it set to `=y` in your config file and that you don't get any warnings. How are you building?

Reply:
With these changes cherry picked to NCS, and `default y if BT_PATH_LOSS_MONITORING` removed, build the zephyr shell with `west build -b nrf52840dk/nrf52840 -p -- -DCONFIG_BT_PATH_LOSS_MONITORING=y` and can see `CONFIG_BT_PATH_LOSS_MONITORING=y` in the .config file but not `CONFIG_BT_CTLR_LE_PATH_LOSS_MONITORING=y`. With no warning printed.

If I do `west build -b nrf52840dk/nrf52840 -p -- -DCONFIG_BT_PATH_LOSS_MONITORING=y -DCONFIG_BT_CTLR_LE_PATH_LOSS_MONITORING=y` then I get the warning.

Reply:
Weird. There really should be a warning if you request a Kconfig value that isn't applied. Sounds like a bug somewhere, and shouldn't be fixed by this approach IMO

Reply:
Looks like we use `select` elsewhere in the file, so have updated to do this